packages/docs/docs/graphql/mutations.md

```diff
@@ -9,11 +9,14 @@ import TabItem from '@theme/TabItem';
 
 # Mutations
 
-GraphQL mutations are operations that allow the client to create, update, or delete data on the server. Unlike queries, which are read-only operations and can be executed in parallel, mutations are write operations. 
+GraphQL mutations are operations that allow the client to create, update, or delete data on the server. Unlike queries, which are read-only operations and can be executed in parallel, mutations are write operations. For more information about GraphQL mutations, refer to the [GraphQL documentation](https://graphql.org/learn/queries/#mutations).
 
-For the inputs, you would append the action (Create, Update, or Delete) to the resource type. Here are examples of mutations for the `Patient` resource:
+Medplum implements the draft [FHIR GraphQL Mutation spec](https://hl7.org/fhir/R4/graphql.html#mutations). For the inputs, you would append the action (Create, Update, or Delete) to the resource type.
+
+Here are examples of mutations for the `Patient` resource:
```

Comment:
High level this looks good.  Some indicaiton on how to use/test this in graphiql would be useful here.